src/features/defi/providers/idle/components/IdleManager/Claim/components/Confirm.tsx

```diff
@@ -141,21 +147,43 @@ export const Confirm = ({ accountId, onNext }: ConfirmProps) => {
     )
   }, [opportunityData?.rewardAssetIds, opportunityData?.rewardsAmountsCryptoBaseUnit])
 
-  const claimableAssets = useMemo(() => {
+  const claimAmounts = useMemo(() => {
     if (!opportunityData?.rewardsAmountsCryptoBaseUnit?.length) return null
 
     return opportunityData?.rewardsAmountsCryptoBaseUnit.map((amount, i) => {
       if (!opportunityData?.rewardAssetIds?.[i]) return null
-
+      const cryptoAmount = bnOrZero(amount)
```

Comment:
Precision vernacular 🙏🏽

Reply:
Actually disregard me, this isn't a precision, it's a human amount with a loss of precision, I'd either use the `precision` vernacular or probably better `number` which carries the lack of very floaty precision